torchaudio/backend/sox_io_backend.py

```diff
@@ -83,37 +112,49 @@ def load(
 @_mod_utils.requires_module('torchaudio._torchaudio')
 def save(
         filepath: str,
-        tensor: torch.Tensor,
```

Comment:
Probably we want to revert these signature changes now that 0.6.0 is built.

Reply:
good catch, do you want to go ahead and just edit the change them directly?